CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # paranoia Changelog
 
+## Unreleased
```

Comment:
I've done a bad job of keeping this up to date. Normally what I would do is just point people at the changes between tags.